docs/configuration.md

```diff
@@ -97,6 +97,18 @@ storage:
 		poolname: radospool
 		username: radosuser
 		chunksize: 4194304
+	swift:
+		username: username
+		password: password
+		authurl: https://storage.myprovider.com/v2.0 or https://storage.myprovider.com/v3/auth
```

Comment:
@nevermosby yes, we are connecting the registry to SoftLayer Object Storage using just authurl, username and password, so I think it is v1, thought I can't find SoftLayer documentation specifically stating that. The push/pull work fine, pending those error messages that I entered in another comment below.